docs/tutorials/basic_commands.md

```diff
@@ -1,23 +1,19 @@
 # Getting Started with Ubuntu Pro Client
 
-The Ubuntu Pro Client (`pro`) provides users with a simple mechanism to
-view, enable, and disable offerings from Canonical on their system. In this tutorial
-we will cover the base `pro` commands that allow a user to successfully manage the offering
-on their machine.
+The Ubuntu Pro Client (`pro`) provides a simple mechanism for viewing, enabling and
+disabling Canonical offerings on your system. In this tutorial we will cover the base
+`pro` commands that will allow you to successfully manage the offerings on your machine.
 
 ## Prerequisites
 
-On this tutorial, you will use [LXD](https://linuxcontainers.org/lxd/) containers.
-To set up LXD on your computer, please follow this [guide](https://linuxcontainers.org/lxd/getting-started-cli/).
-
-```{tip}
-When configuring LXD on your machine, run the `lxd init --minimal`. This should be more
-than enough for you to finish this tutorial
-```
+In this tutorial, you will use [Multipass](https://multipass.run/) virtual machines.
```

Comment:
As a "new user", I was left feeling a bit confused. If the pro tool allows us to manage the offerings on our machine, why do we need to use a VM? I think it would be helpful to add a brief explanation to explain the context - firstly why we would want to use a VM in the first place, and secondly why we want to use Multipass.

Reply:
I'm still not 100% clear on why we're using a VM. Is there some reason why we should use one and not just run the commands on our machine directly?